Calendar/Get-RBASummary.ps1

```diff
@@ -273,14 +272,24 @@ function RBAProcessingValidation {
 }
 
 # Write out a list of Mailboxes
+# We get CN from the cmdlet and want Display Name and Primary SMTP Address
 function OutputMBList {
     param (
         [Parameter(Mandatory)]
         [string[]]$MBList
     )
     foreach ($User in $MBList) {
-        $User = Get-Mailbox -Identity $User
-        Write-Host " `t `t [$($User.DisplayName)] -- $($User.PrimarySmtpAddress)"
+        # MS Support will error as we need the Organization to process from CN
+        if ($Identity -like "*@*") {
+            $Org = $Identity.Split('@')[1]
+        }
+        if ($null -ne $Org) {
```

Comment:
Bug, need to set `$Org` to `$null` each time in the loop, otherwise `$Org` could be the last `$Identity` in the `$MBList`. 

You should be able to also do this: 
```
# No need for 'if ($Identity -like "*@*")' as split() doesn't return an error
$Org = $Identity.Split("@")[1]
if ($null -ne $Org) {...
```

This way `$Org` is always set correctly in the loop.

Reply:
@Shanefe just sending out a reminder.